src/data-loader.ts

```
@@ -10,64 +10,153 @@
+			.update(JSON.stringify(relevantOptions, Object.keys(relevantOptions).sort()))
+			.digest('base64');
+	}
+
+	public async getCacheDir(): Promise<string> {
+		const cacheDir = path.join(tmpdir(), 'ccusage-cache');
+		await mkdir(cacheDir, { recursive: true });
+		return cacheDir;
+	}
+
+	public loadCache(filePath: string): ReturnType<typeof createLimoJson<CacheSchema | undefined>> {
+		const cache = createLimoJson<CacheSchema | undefined>(filePath);
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)